packages/splitter/src/SplitterContainer.spec.tsx

```diff
@@ -154,32 +182,41 @@ describe('SplitterContainer', () => {
   describe('getSeparatorProps', () => {
     it('returns correct default props', () => {
       let separatorProps;
+      let primaryPaneProps;
 
       render(
         <SplitterContainer
           ariaLabel="flex-pane"
-          primaryPaneId="flex-pane"
           type={SplitterType.VARIABLE}
           min={0}
           max={100}
           orientation={SplitterOrientation.VERTICAL}
+          placement={SplitterPlacement.END}
         >
-          {({ getSeparatorProps }: IUseSplitterReturnValue) => {
+          {({ getSeparatorProps, getPrimaryPaneProps }: IUseSplitterReturnValue) => {
             separatorProps = getSeparatorProps();
+            primaryPaneProps = getPrimaryPaneProps();
 
             return null;
           }}
         </SplitterContainer>
       );
+      expect(primaryPaneProps).toMatchInlineSnapshot(`
+        Object {
+          "data-garden-container-id": "containers.splitter.primaryPane",
+          "data-garden-container-version": "version",
+          "id": "1",
+        }
+      `);
```

Comment:
What's the preference for using a snapshot here?

Reply:
Its a way to verify the signature of `getPrimaryPaneProps` and `getSeparatorProps` outputs beyond type checking (value checking in this case).